src/sagemaker_xgboost_container/data_utils.py

```diff
@@ -238,13 +255,15 @@ def validate_data_file_path(data_path, content_type):
             data_files = [
                 os.path.join(dir_path, file_name) for file_name in os.listdir(dir_path) if _is_data_file(
                     dir_path, file_name)]
-
         if parsed_content_type.lower() == CSV:
             for data_file_path in data_files:
                 _validate_csv_format(data_file_path)
         elif parsed_content_type.lower() == LIBSVM:
             for data_file_path in data_files:
                 _validate_libsvm_format(data_file_path)
+        elif parsed_content_type.lower() == PARQUET or parsed_content_type.lower() == RECORDIO_PROTOBUF:
+            # No op
+            return
```

Comment:
Would a user of script mode ever use this method? I can add an else just in case, but in algo mode we only pass in a validated content type to this method.

Reply:
Actually, it looks like get_content_type() is called at the top of the function, so we're guaranteed that only valid content types will reach this part of the code. An else statement is not necessary.